elastic/security/challenges/security-indexing-querying.json

```diff
@@ -31,6 +31,9 @@
                 "operation-type": "composite",
                 "param-source": "workflow-selector",
                 "workflow": {{workflow | tojson }},
+                {% if p_index_mode == "logsdb" %}
```

Comment:
Shouldn't this be determined by the `work_flow` track param? Or is the idea that the `work_flow` param is determined by the index mode track param?

Reply:
the workflow parameter just says hosts, network or overview - the workflows-folder is what refers to either currently workflows, but for logsdb we want to use the workflows-logsdb folder